### DISCUSSION TITLE
Embedding videos?

### DISCUSSION THREAD

**Discussion by MehmetMHY** [hugo-sid/hugo-blog-awesome#77] [General]
I was wondering if there is anyway to embed a video for a post? I know that in markdown you can add a link or make an image into a link. But I was wondering if I could view a video inside a post rather then go to a new tab.

**Comment by MehmetMHY** [comment]
Sorry, I just figured out how to achieve this. So lets say you want to embed this youtube video: https://www.youtube.com/watch?v=QlRtFugjQtw

Grab the string after "v=", which in this case is "QlRtFugjQtw". Then add this to your markdown file:
```
{{< youtube QlRtFugjQtw >}}
```

This should embed a YouTube video into your post.

**Comment by hugo-sid** [comment]
You are right @MehmetMHY. That's the correct usage of YouTube shortcode.
